### PR TITLE
docs: fix documentation errors in RFS,EVS,SMS,FGS,MRS

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:
fix documentation errors in RFS,EVS,SMS,FGS,MRS
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

```
====== Checking for resources huaweicloud_rfs_stack =====
the format of `agency.provider_name` is not correct, should be (Optional, String, ForceNew)
the format of `agency.name` is not correct, should be (Optional, String, ForceNew)

====== Checking for resources huaweicloud_evs_volume =====
can not find `attachment.instance_id`, please check it
can not find `attachment.device`, please check it

====== Checking for resources huaweicloud_sms_task =====
the format of `target_server_disks.size` is not correct, should be (Required, Int, ForceNew)
the format of `target_server_disks.used_size` is not correct, should be (Optional, Int, ForceNew)
the format of `target_server_disks.physical_volumes.used_size` is not correct, should be (Optional, Int, ForceNew)
the format of `target_server_disks.physical_volumes.size` is not correct, should be (Required, Int, ForceNew)

====== Checking for resources huaweicloud_fgs_function =====
the format of `custom_image` is not correct, should be (Optional, List)
the format of `custom_image.url` is not correct, should be (Required, String)
can not find `network_id`, please check it
can not find `vpc_id`, please check it
the format of `app` is not correct, should be (Optional, String)

====== Checking for resources huaweicloud_mrs_cluster =====
can not find `internal_ip`, please check it
```

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```

```

### CODE REVIEW

File: huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go [hunk 1]
```diff
@@ -84,12 +84,13 @@ func ResourceFgsFunctionV2() *schema.Resource {
 				Type:          schema.TypeString,
 				Optional:      true,
 				ConflictsWith: []string{"app"},
-				Deprecated:    "use app instead",
+				Deprecated:    "schema: Internal; use app instead",
```
Comment (ShiChangkuo) [inline huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go]: it's unnecessaty to change this as using **Deprecated** just be fine.

File: docs/resources/fgs_function.md [hunk 2]
```diff
@@ -158,9 +158,8 @@ The following arguments are supported:
 * `func_mounts` - (Optional, List) Specifies the file system list. The `func_mounts` object structure is documented
   below.
 
-* `custom_image` - (Optional, List, ForceNew) Specifies the custom image configuration for creating function.
+* `custom_image` - (Optional, List) Specifies the custom image configuration for creating function.
```
Comment (Lance52259) [inline docs/resources/fgs_function.md:161]: The update method doesn't check this parameter.

Comment (ShiChangkuo) [inline docs/resources/fgs_function.md:161]: should add `custom_image` in HasChanges list

**Comment by deer-hang** [comment]
```
make testacc TEST='./huaweicloud/services/acceptance/fgs' TESTARGS='-run TestAccFgsV2Function_createByImage'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/fgs -v -run TestAccFgsV2Function_createByImage -timeout 360m -parallel 4 
=== RUN   TestAccFgsV2Function_createByImage 
=== PAUSE TestAccFgsV2Function_createByImage
=== CONT  TestAccFgsV2Function_createByImage
--- PASS: TestAccFgsV2Function_createByImage (67.26s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/fgs       67.309s
```

**Pull request merged**